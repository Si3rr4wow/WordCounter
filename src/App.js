import React from 'react'

import { Container } from 'reactstrap'
import { FileCounter, InputCounter } from 'Components'

const App = () => {
  return (
    <Container>
      <h1>
        I&apos;m a handy tool for counting words
      </h1>
      <InputCounter/>
      <hr/>
      <FileCounter/>
    </Container>
  )
}

export default App
