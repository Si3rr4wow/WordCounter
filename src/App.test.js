import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

it('Renders the app', () => {
  const tree = renderer
    .create(<App/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
