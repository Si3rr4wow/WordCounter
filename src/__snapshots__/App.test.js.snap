// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the app 1`] = `
<div
  className="container"
>
  <h1>
    I'm a handy tool for counting words
  </h1>
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      <h4>
        Use me to analyse smaller bodies of text.
      </h4>
    </div>
    <div
      className="card-body"
    >
      <p>
        I can count the words you type here synchronously
      </p>
      <textarea
        className="form-control"
        onChange={[Function]}
        value=""
      />
    </div>
  </div>
  <hr />
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      <h4>
        Use me if you've got a larger file to analyse.
      </h4>
    </div>
    <div
      className="card-body"
    >
      <p>
        I spin up web workers to count the words in the file you pass me. This means that the event loop doesn't get clogged up during the long running process.
      </p>
      <input
        className="form-control-file"
        disabled={false}
        onChange={[Function]}
        type="file"
      />
    </div>
  </div>
</div>
`;
